velox/connectors/hive/tests/HivePartitionFunctionTest.cpp

```diff
@@ -277,3 +278,17 @@ TEST_F(HivePartitionFunctionTest, date) {
   assertPartitionsWithConstChannel(values, 500);
   assertPartitionsWithConstChannel(values, 997);
 }
+
+TEST_F(HivePartitionFunctionTest, HivePartitionFunctionSpecTest) {
+  int bucketCount = 10;
+  std::vector<int> bucketToPartition(bucketCount);
+  std::iota(bucketToPartition.begin(), bucketToPartition.end(), 0);
+  std::vector<column_index_t> channels{1};
+  std::vector<VectorPtr> constValues{nullptr};
+  auto hivePartitionFunctionSpec =
+      std::make_unique<connector::hive::HivePartitionFunctionSpec>(
+          bucketCount, bucketToPartition, channels, constValues);
+  ASSERT_EQ(
+      hivePartitionFunctionSpec->toString(),
+      "HIVE({NUM_BUCKETS:10, KEYS channels:1, KEYS constValues:})");
```

Comment:
Can we add tests with non-empty const values and multiple channels?

This string is not very human friendly. How about "Hive(buckets: 10, channels: 1)"?

Can we also add serde tests?